internal/config/database.go

```diff
@@ -11,11 +12,12 @@ import (
 )
 
 type database struct {
-	Dir                   string `yaml:"cache-dir" json:"cache-dir" mapstructure:"cache-dir"`
-	UpdateURL             string `yaml:"update-url" json:"update-url" mapstructure:"update-url"`
-	CACert                string `yaml:"ca-cert" json:"ca-cert" mapstructure:"ca-cert"`
-	AutoUpdate            bool   `yaml:"auto-update" json:"auto-update" mapstructure:"auto-update"`
-	ValidateByHashOnStart bool   `yaml:"validate-by-hash-on-start" json:"validate-by-hash-on-start" mapstructure:"validate-by-hash-on-start"`
+	Dir                   string        `yaml:"cache-dir" json:"cache-dir" mapstructure:"cache-dir"`
+	UpdateURL             string        `yaml:"update-url" json:"update-url" mapstructure:"update-url"`
+	CACert                string        `yaml:"ca-cert" json:"ca-cert" mapstructure:"ca-cert"`
+	AutoUpdate            bool          `yaml:"auto-update" json:"auto-update" mapstructure:"auto-update"`
+	ValidateByHashOnStart bool          `yaml:"validate-by-hash-on-start" json:"validate-by-hash-on-start" mapstructure:"validate-by-hash-on-start"`
+	StalenessThreshold    time.Duration `yaml:"staleness-threshold" json:"staleness-threshold" mapstructure:"staleness-threshold"`
```

Comment:
It's not immediately obvious what "staleness threshold" means. We should be exact here. We want this value to represent the max time allowed since the database was built.

Reply:
I agree, I renamed it to `MaxAllowedDBAge`, which I think is more explicit and direct. Please let me know of any possible redundancy.